backend/app/platforms/schemas.py

```diff
@@ -0,0 +1,30 @@
+import json
+from uuid import UUID
+
+from pydantic import field_validator
+
+from app.shared.schema import ORMModel
+
+
+class Identifiers(ORMModel):
+    identifiers: list[str]
+
+
+class PlatformServiceConfigSchema(ORMModel):
```

Comment:
This is a temp config I pressume? You will probably create S3 specific and Glue specific schemas in the future?

Reply:
No, this is the schema used to return generic config.

Reply:
Identifiers might be converted to something specific later

Reply:
I think we are talking about the same thing :D
Fine for now as a first implementation